data/phi0474/phi031/phi0474.phi031.perseus-lat1.xml

```diff
@@ -678,12 +699,14 @@ Added to repository. Passes SGML validation.
 				venit nisi uter utri insidias fecerit? <reg>profecto</reg> nihil: si hic illi, ut ne
 				sit impune; si ille huic, tum nos<note>tum nos <hi rend="italics">ET<foreign
 							xml:lang="grc">d</foreign></hi>: ut <hi rend="italics">P</hi></note>
-				scelere solvamur. <milestone n="32" unit="section"/></p>
-			<p><reg>quonam</reg> igitur pacto probari potest insidias Miloni fecisse Clodium?
+				scelere solvamur. </p>
+	</div>
+	<div n="32" subtype="section" type="textpart">
+			<p rend="indent"><reg>quonam</reg> igitur pacto probari potest insidias Miloni fecisse Clodium?
 					<reg>satis</reg> est in illa quidem tam audaci, tam nefaria belua docere, magnam
 				ei causam, magnam spem in Milonis morte propositam, magnas utilitates fuisse.
-					<reg>itaque</reg> illud Cassianum 'cui bono fuerit<note><lemma>cui bono
-						fuerit</lemma> iudicium (indicium <hi rend="italics">Purser</hi>) <hi
+					<reg>itaque</reg> illud Cassianum 'cui bono fuerit<note><w type="lemma">cui bono
```

Comment:
quotes need to be captured. `<q>` for simple speech, emphasis, inline stuff and `<quote>` for quotations (mainly block quotes)

quotes that span its containers employ rend="merge" to reflect the continuation